--- openscad.pro.orig	2019-02-23 11:50:06.000000000 -0800
+++ openscad.pro	2019-04-25 18:44:14.000000000 -0700
@@ -96,7 +96,6 @@
   APP_RESOURCES.files = OpenSCAD.sdef dsa_pub.pem icons/SCAD.icns
   QMAKE_BUNDLE_DATA += APP_RESOURCES
   LIBS += -framework Cocoa -framework ApplicationServices
-  QMAKE_MACOSX_DEPLOYMENT_TARGET = 10.9
 }
 
 # Set same stack size for the linker and #define used in PlatformUtils.h
